src/index.js

```diff
@@ -82,7 +83,27 @@ app.post('/webhooks/bundestagio/update', async (req, res) => {
   } catch (error) {
     console.log(error);
     res.send({
-      error,
+      error: error.message,
```

Comment:
@ReggaePanda Error field was always empty `{ errror: {}, ... }`. Usually this is covered with a software test. Any plans to setup a test runner, e.g. mocha?